docs/Rest-API.md

```diff
@@ -0,0 +1,147 @@
+# Axe REST API
+
+Documentation of the REST API that comes along with the Axe service.
```

Comment:
>REST API [](start = 21, length = 8)

What REST verb is being documented here? It feels like GET, but the docs don't say. Also, what parameters, if any, should be specified by the request? For example, is the scan always the entire display or just part of it? What about multi-screen devices where different screens could potentially have different apps/views? The Android documentation says that Android doesn't support a "root" or "desktop" view, so would multiple apps require a collection of separate hierarchies?